src/clojure/clj_ldap/client.clj

```diff
@@ -44,6 +45,21 @@
 (defn encode ^String [^Attribute attr]
   (.getValue attr))
 
+(defn ssl-protocol-mapping
+  "Converts user protocol settings into valid string constants"
+  [ssl-protocols]
+  (let [valid-protocols #{"TLSv1.3" "TLSv1.2" "TLSv1.1" "TLSv1"}
+        valid-ssl-protocols (filter #(contains? valid-protocols %) ssl-protocols)
+        _invalid-ssl-protocols (remove #(contains? valid-protocols %) ssl-protocols)]
+    (when not-nil? _invalid-ssl-protocols
+      (log/infof "Unsuported value %s passed into SSL protocol" _invalid-ssl-protocols)
+      (log/infof "Removing %s from ssl-protocol" _invalid-ssl-protocols))
+    (replace 
+     {"TLSv1.3" "SSL_PROTOCOL_TLS_1_3" 
```

Comment:
``suggestion
     {"TLSv1.3" SSLUtil/SSL_PROTOCOL_TLS_1_3
      "TLSv1.2" SSLUtil/SSL_PROTOCOL_TLS_1_2,
      "TLSv1.1" SSLUtil/SSL_PROTOCOL_TLS_1_1,
      "TLSv1" SSLUtil/SSL_PROTOCOL_TLS_1} valid-ssl-protocols)))
```
We should prefer the symbol declared in the class rather than assuming it maps to the same string.  This syntax should allow us to use the static value in the class itself.